core/src/test/java/org/apache/iceberg/io/InMemoryInputFile.java

```diff
@@ -64,6 +64,7 @@ private static class InMemorySeekableInputStream extends SeekableInputStream {
 
     private final int length;
     private final ByteArrayInputStream delegate;
+    private boolean closed = false;
```

Comment:
Since these changes are mostly unrelated, can you separate them into their own PR?

Reply:
sure: https://github.com/apache/iceberg/pull/5019